queries/luap/highlights.scm

```diff
@@ -32,7 +32,10 @@
 (negated_set
   (character) @constant)
 
-(class) @keyword
+(class) @string.escape
+
+(class
+  (escape_char)) @string.luap
```

Comment:
Why not just 
```suggestion
(class
  (escape_char) @string.escape)
```

Reply:
In any case, `@string.luap` is not an allowed capture name.

Reply:
I use that capture because I was trying to do something similar to this:

https://github.com/nvim-treesitter/nvim-treesitter/blob/a2d6678bb21052013d0dd7cb35dffbac13846c98/queries/regex/highlights.scm#L20

Reply:
That `regexp` is not the language name, though (which is `regex`, in fact).

Reply:
Oh, honestly didn't know. Should I use that capture instead? Since the `(escape_char)` node is similar to `(identity_escape)`

Reply:
Yes, that's exactly what @lucario387 suggested.

Reply:
He suggested using `@string.escape`, not `@string.regexp`

Reply:
Well it's an escape character, I'd expect the result to be either literal string or escaped strings, not a regex/lua pattern

Reply:
`(escape_char)` is for a escaped special char (`.` -> `%.`) and since the equivalent in regex (`.` -> `\.`) is captured as `@string.regexp` I'd expect luap to do the same

Reply:
So I change it to `@string.escape` or leave it as `@string.regexp`?